recipes/metasnv/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+export CFLAGS="-I$PREFIX/include"
+export CPPFLAGS="-I$PREFIX/include"
+export LDFLAGS="-L$PREFIX/lib"
+tag=share/metasnv-1.0.1/
```

Comment:
you can use variables like `$PKG_VERSION` here,
https://conda.io/docs/user-guide/tasks/build-packages/environment-variables.html

Reply:
Thanks, I'll update that